response_operations_ui/views/collection_exercise.py

```diff
@@ -1147,12 +1150,32 @@ def view_sample_ci_summary(short_name: str, period: str) -> str:
 @collection_exercise_bp.route("/<short_name>/<period>/view-sample-ci/summary/<form_type>", methods=["GET"])
 @login_required
 def view_ci_versions(short_name: str, period: str, form_type: str) -> str:
+    survey_id = survey_controllers.get_survey_by_shortname(short_name).get("id")
```

Comment:
Not sure of the property, but `.get("id")` might need to be `.get("survey_id")` or something similar?

Reply:
Updated to be survey_ref.